pkg/instancegroups/rollingupdate.go

```diff
@@ -160,6 +163,21 @@ func (c *RollingUpdateCluster) RollingUpdate(groups map[string]*cloudinstances.C
 		}
 	}
 
+	// Upgrade API servers
+	{
+		for k := range apiServerGroups {
+			results[k] = fmt.Errorf("function panic nodes")
```

Comment:
```suggestion
			results[k] = errors.New("function panic nodes")
```
That error message is confusing.

Reply:
Error is copypasted from the other roles